tests/framework/matcher/existence.go

```diff
@@ -31,12 +32,37 @@ func (e existMatcher) Match(actual interface{}) (success bool, err error) {
 	return true, nil
 }
 
+func formatObject(actual interface{}) string {
+	obj := reflect.ValueOf(actual)
+	if !helper.IsStruct(actual) {
```

Comment:
Stupid question, but does it also work if `actual` is a ptr to a struct? Not sure that is required though.

Reply:
It should now.

Reply:
Won't `helper.ToPointer` break the check `helper.IsStruct` below? `helper.ToPointer` sets the kind of `actual` to `reflect.Ptr`, but `helper.IsStruct` expects it to be `reflect.Struct`?

If I read [this](https://pkg.go.dev/reflect#Value.Elem) correctly, then it is fine to pass in either an interface value or a pointer, so we don't need to convert again? I'm unsure about it however.

Reply:
Reworked